extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -55,14 +49,14 @@ public function executeIndex() {
 			$this->returnTo = $this->wg->Script;
 		}
 
-		$wgOut->getResourceLoader()->getModule( 'mediawiki' );
+		$this->wg->Out->getResourceLoader()->getModule( 'mediawiki' );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)